ci-operator/jobs/openshift/builder/openshift-builder-master-postsubmits.yaml

```diff
@@ -14,6 +14,7 @@ postsubmits:
         - --give-pr-author-access-to-namespace=true
         - --artifact-dir=$(ARTIFACTS)
         - --target=[images]
+        - --promote
```

Comment:
This can't be promoted yet. But ci-operator doesn't allow you to indicate "no promotion" right now.  We could add that to ci-operator.

Reply:
What's the blocker?